ArduSub/inertia.cpp

```diff
@@ -4,18 +4,21 @@
 void Sub::read_inertia()
 {
     // inertial altitude estimates
-    inertial_nav.update(G_Dt);
+    inertial_nav.update();
 
-    // pull position from interial nav library
-    current_loc.lng = inertial_nav.get_longitude();
-    current_loc.lat = inertial_nav.get_latitude();
+    // pull position from ahrs
+    Location loc;
+    ahrs.get_position(loc);
+    current_loc.lat = loc.lat;
+    current_loc.lng = loc.lng;
 
     // exit immediately if we do not have an altitude estimate
     if (!inertial_nav.get_filter_status().flags.vert_pos) {
         return;
     }
 
-    current_loc.alt = inertial_nav.get_altitude();
+    current_loc.set_alt_cm(inertial_nav.get_altitude(),
```

Comment:
I can revert this, it was originally because I was just doing a full set of `current_loc`. With that being said though I'm pretty sure we are slowly trying to push people into always using the `set_alt_cm` methods to fix frame handling problems. But if it's holding it up, I'll revert this.